src/main/kotlin/com/exactpro/th2/common/schema/message/MessageRouterUtils.kt

```diff
@@ -71,4 +74,16 @@ fun appendAttributes(
     return mutableSetOf(*attributes).apply {
         addAll(requiredAttributes())
     }
+}
+
+fun MessageGroupBatch.toShortDebugString(): String = buildString {
+    append("MessageGroupBatch(ids = ")
+
+    groupsList.asSequence()
+        .flatMap { it.messagesList.asSequence() }
+        .map(AnyMessage::logId)
+        .toSortedSet()
```

Comment:
Why sorted set? To sort it by session alias? Because we will sort the long ID and I don't think that sorting will perform the way you expect

Reply:
> To sort it by session alias?

Yeah.

> I don't think that sorting will perform the way you expect

Why not?

Reply:
My bad, the colon will always be lower than any letter. Will work fine, I guess.